content/en/guide/v10/components.md

```diff
@@ -40,6 +41,7 @@ Class components can have state and lifecycle methods. The latter are special me
 Here we have a simple class component called `<Clock>` that displays the current time:
 
 ```jsx
+// --repl
```

Comment:
```suggestion
// --repl
import { Component, render } from 'preact';

// --repl-before
```

We'll also need the `-after` but GitHub is annoying and won't you suggest on unchanged lines:

```
// --repl-after
render(<Clock />, document.body);
```